aclpub2/paper_csv_to_yaml.py

```diff
@@ -0,0 +1,46 @@
+import csv
+import yaml
+import collections
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--csv_path", 
+        type=str, 
+        help="Path to input csv."
+)
+parser.add_argument(
+        "--out_path",
+        type=str,
+        default="output.yaml",
+        help="Name of YAML file to write to.",
+)
+parser.add_argument("--author_field_name", 
+        type=str, 
+        default="Authors_Full"
+)
+args = parser.parse_args()
+
+
+all_entries = []
+with open(args.csv_path, 'r') as f:
+    reader = csv.reader(f, delimiter=',')
+    header = next(reader)
+    for row in reader:
+        row_out = collections.OrderedDict()
+        authors = collections.defaultdict(dict)
+        for k, v in zip(header, row):
```

Comment:
@cimeister could you change this to use https://docs.python.org/3/library/csv.html#csv.DictReader ? it'll make it slightly more readable I think.

Reply:
good idea!